sklearn/model_selection/_search.py

```diff
@@ -704,19 +757,51 @@ class GridSearchCV(BaseSearchCV):
            fit_params={}, iid=..., n_jobs=1,
            param_grid=..., pre_dispatch=..., refit=...,
            scoring=..., verbose=...)
-
+    >>> sorted(clf.results_.keys())
+    ...                             # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+    ['param_C', 'param_kernel', 'params', 'test_mean_score',...
+     'test_rank_score', 'test_split0_score', 'test_split1_score',...
+     'test_split2_score', 'test_std_score']
 
     Attributes
     ----------
-    grid_scores_ : list of named tuples
-        Contains scores for all parameter combinations in param_grid.
-        Each entry corresponds to one parameter setting.
-        Each named tuple has the attributes:
-
-            * ``parameters``, a dict of parameter settings
-            * ``mean_validation_score``, the mean score over the
-              cross-validation folds
-            * ``cv_validation_scores``, the list of scores for each fold
+    results_ : dict of numpy (masked) ndarrays
+        A dict with keys as column headers and values as columns, that can be
+        imported into a pandas DataFrame.
+
+        For instance the below given table
+
+        param_kernel|param_gamma|param_degree|test_split0_score|...|...rank...|
+        =======================================================================
+        'poly'      |      -    |      2     |        0.8      |...|    2     |
+        'poly'      |      -    |      3     |        0.7      |...|    4     |
+        'rbf'       |     0.1   |      -     |        0.8      |...|    3     |
+        'rbf'       |     0.2   |      -     |        0.9      |...|    1     |
+
+        will be represented by a results_ dict of :
```

Comment:
double backticks around `results_`, and put the code snippet in an appropriate RST environment